app/jobs/fill_flagged_cache_job.rb

```diff
@@ -0,0 +1,8 @@
+class FillFlaggedCacheJob < ApplicationJob
+  queue_as :default
+
+  # fills the cache - used by app/views/users/show.html.erb for mods
```

Comment:
Not sure if there's a comment convention for ActiveJobs, nor whether y'all wanna keep this comment from the old script

Reply:
I don't know of a formatting convention, but I appreciate you keeping it for context. These informal little cross-references help me a lot.